.github/workflows/ci_pipeline.yml

```diff
@@ -65,20 +97,13 @@ jobs:
           fail_ci_if_error: true
           verbose: true
```

Comment:
Should we add a condition to upload documentation artifacts and code coverage reports in only one of the jobs?

Something like:
```yaml
- if: ${{ matrix.ros_distro == 'humble' }}
  name: Upload documentation artifacts
```

Reply:
Or better yet, do something like what is proposed in https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations.

```yaml
strategy:
      fail-fast: false
      matrix:
        ros_distro:
          - noetic
          - humble
      include:
          - ros_distro: humble
            upload_artifacts: true
```

```yaml
- if: ${{ matrix.upload_artifacts }}
  name: Upload documentation artifacts
  
- if: ${{ matrix.upload_artifacts }}
  name: Upload code coverage report
```

Reply:
OK with uploading documentation artifacts once, but it does make sense to have coverage reports in Noetic as well. Not sure how to configure `codecov` for that though.